traffic_ops/traffic_ops_golang/topology/topologies.go

```diff
@@ -394,8 +394,10 @@ func (topology TOTopology) GetKeys() (map[string]interface{}, bool) {
 
 // SetKeys is a requirement of the api.Updater interface and is called by
 // api.UpdateHandler().
+// SetKeys function will no longer makes the name key immutable by setting it to
+// query param. On 2/3/21, the requirement was added to change the topology name.
 func (topology *TOTopology) SetKeys(keys map[string]interface{}) {
-	topology.Name, _ = keys["name"].(string)
+	//topology.Name, _ = keys["name"].(string)
```

Comment:
So, I'm not positive, but I _think_ the `Validation()` method might be doing validation based on the _requested_ name, rather than the _current_ name, which would be bad. Can you double-check if that's happening or not?

Reply:
Good catch.. I updated the Validate()